models/index.js

```diff
@@ -0,0 +1,43 @@
+const Sequelize = require('sequelize')
+const db = new Sequelize('postgres://localhost:5432/authenticate',{logging:false})
+
+const User = db.define('user',{
+    name: {
+        type:Sequelize.STRING,
+        allowNull:false
+    },
+    email:{
+        type:Sequelize.STRING,
+        allowNull:false,
+        validate: {
+            isEmail:true
+        },
+        unique:true
+    },
+    phone:{
+        type:Sequelize.STRING,
+        allowNull:false,
+        unique:true
+    },
+    authenticated:{
+        type:Sequelize.BOOLEAN,
+        allowNull:false,
+        defaultValue:false
+    }
+
+})
+const Code = db.define('code',{
+    authCode:{
```

Comment:
standardize and use camel OR snake case.  I'm not sure the standard if the db is accessed via Node, if it then goes to camel case, so be it.   I've always used snakecase backend languages and have seen db tables use snake case.  In either case, choose one